### PR TITLE
Compute values for chart to be installed

### PR DESCRIPTION
**Pull request by jsravn**
For https://github.com/lightbend/console-backend/issues/606

### CODE REVIEW

File: enterprise-suite/scripts/lbc.py [hunk 1]
```diff
@@ -76,10 +79,6 @@ def fail(msg):
     sys.exit(msg)
 
 
-def make_tempdir():
```
Comment (mitkus) [inline enterprise-suite/scripts/lbc.py]: Some tests override this function, they will have to be changed if you remove it

Comment (jsravn) [reply enterprise-suite/scripts/lbc.py]: Yes discovered this when the tests failed...

**Comment by jsravn** [comment]
Ready for re-review.

**Comment by jsravn** [comment]
This is an implementation detail of the PVC work which still needs some good tests - tbd in https://github.com/lightbend/console-backend/issues/608.

**Comment by jsravn** [comment]
Do you mean stubbing out a method? I just find doing that kind of testing is making the maintainability of the install script already quite difficult. I think we need more black box tests for this kind of thing. I will think about it some more.

**Comment by jsravn** [comment]
Unfortunately the `template` command fails if you pass in all the install flags. So I need to prune the flags down to just those that template can use (--set*, --values).

File: enterprise-suite/scripts/lbc.py (hunk 1 above)
Comment (jsravn) [reply enterprise-suite/scripts/lbc.py]: Replaced with `make_fetchdir` farther down in the file.

**Pull request merged**